setup.py

```diff
@@ -21,11 +28,13 @@
     url='https://gitlab.com/rbraswell/multitemporal.git',
     author='Bobby H. Braswell',
     author_email='rbraswell@ags.io',
-    classifiers=[
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Cython :: 0.26',
-    ],
+    # it's not clear what classifiers are appropriate and anyway it only matters for pypi:
+    #classifiers=[
+    #    'Programming Language :: Python :: 2.7',
+    #    'Programming Language :: Cython :: 0.26',
+    #],
     license='GPLv3',
+    python_requires='>=3',
```

Comment:
This line is roughly harmless as probably there aren't any mt users that are bound to python 2, but if more commits get added to this PR, consider taking it out (whole reason it's here is because I forgot futurize defaults to dual support for both python 2 and 3).